spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/domain/PackageSummary.java

```diff
@@ -40,4 +40,6 @@
 	String getIconUrl();
 
 	String getDescription();
+
+	String getRepositoryName();
```

Comment:
The search command can now display this as well. Created https://github.com/spring-cloud/spring-cloud-skipper/issues/451 to track this.